app.js

```diff
@@ -2,6 +2,6 @@ const express = require('express');
 const app = express();
 const port = 3000;
 
-app.get('/', (req, res) => res.send('Hello World!'));
+app.get('/', (req, res) => res.sendFile(path.join(__dirname, './', 'index.html')));
```

Comment:
When I run locally, I get "path is not defined"

"Path" is a library that comes with node, but it still needs to be imported.

Use this line:

`var path = require('path');`